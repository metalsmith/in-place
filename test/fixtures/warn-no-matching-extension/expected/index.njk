<h2 id="title">Title</h2>
<h1>{{ title }}</h1>
