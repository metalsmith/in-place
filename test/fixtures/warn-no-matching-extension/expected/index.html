<h2 id="title">Title</h2>
